torchgeo/datamodules/inria.py

```diff
@@ -98,7 +98,7 @@ def patch_sample(self, sample: Dict[str, Any]) -> Dict[str, Any]:
             self.patch_size,
             padding=padding,
         )
-        sample["image"] = rearrange(sample["image"], "() t c h w -> t () c h w")
+        sample["image"] = rearrange(sample["image"], "b n c h w -> (b n) c h w")
```

Comment:
`extract_tensor_patches` takes in "b c h w" and returns "b n c h w" where n is the number of patches extracted per image. This additional axis causes our code to fail. I think we should merge this into the batch dimension to fix this issue.

See https://kornia.readthedocs.io/en/latest/contrib.html#kornia.contrib.ExtractTensorPatches for details.

Reply:
Could you add this line just after `extract_tensor_patches`?
 
```
sample["num_patches"] = sample["image"].shape[1]
```

This could be then used to reshape it back to "n b c h w" as is required in #560